setup_env.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+./ci/install-protobuf.sh
+
+go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
```

Comment:
you are accessing github.com a lot, i'd put it in a pramaeter $GITHUB

Reply:
I think the readability and code complexity tradeoff isn't worth the DRY-ness